fdbserver/CommitProxyServer.actor.cpp

```diff
@@ -897,6 +903,29 @@ ACTOR Future<Void> getResolution(CommitBatchContext* self) {
 	self->transactionResolverMap.swap(requests.transactionResolverMap);
 	// Used to report conflicting keys
 	self->txReadConflictRangeIndexMap.swap(requests.txReadConflictRangeIndexMap);
+
+	// Fetch cipher keys if needed.
+	state Future<std::unordered_map<EncryptCipherDomainId, Reference<BlobCipherKey>>> getCipherKeys;
+	if (SERVER_KNOBS->ENABLE_TLOG_ENCRYPTION) {
+		std::unordered_map<EncryptCipherDomainId, EncryptCipherDomainName> encryptDomains = {
```

Comment:
a nit: move the definition to `anonymous namespace` as it can be used between the calls?